flows/index.md

```diff
@@ -0,0 +1,128 @@
+- [Step 2: Build the flow](#build-the-flow)
+- [Step 3: Execute tasks in parallel](#execute-tasks-in-parallel)
+- [Step 4: Schedule and automate your flow](#schedule-and-automate)
+- [Step 5: Set up notifications](#set-up-notifications)
+- [Step 6: Check how your flow is running](#check-run-history)
+
+## Access the Flow Builder
+Select from the top menu **Flows -> Flows** and click the **Create Flow** button.
+   
+{: .image-popup}
+![Go to Flows](/tutorial/automate/automate1.png)
+
+Name your new flow and add an easy to understand description. Then click **Create Flow** again. 
+
+This will open up the **flow builder** view, where you can create your data flow.
+
+## Build the Flow
+Click **Select First Step** and start selecting the components that will bring in data from your selected data sources. Use the drop-down menu to select a particular configuration of the component.
+
+{: .image-popup}
+![Select First Step](/tutorial/automate/automate3.png)
+
+Use the plus icon to add other steps. 
+
+Apart from the extractor(s), you'll want to add selected transformations and writers that will load your transformed data into your selected data destination.
+Continue adding other ready-made components and organize them in a logical flow,  or select those you want to run in parallel using drag-and-drop function. 
+
+- Organize the components into a logical flow using the drag-and-drop functionality.
+- Configure each component by providing the credentials and instructions for what or where to extract or write or for what code to execute in a transformation.
+- Remember to adjust the input mapping of your transformation(s) to use the tables extracted from your selected data sources if necessary. Select the step and click **Edit Configuration**.
+- Save the changes.
+
+Once you’ve built your flow end-to-end, it may look something like this:
+
+{: .image-popup}
+![Add SQL Transformation](/tutorial/automate/automate10.png)
+   
+Click **Run Flow** to start the data pipeline.
+
+## Execute Tasks in Parallel
+You can group multiple tasks within one step, a phase. These tasks then run independently in parallel, speeding up the execution. 
+Steps execute sequentially, while tasks within a single step run in parallel. If you have multiple extractors, you can include them all in a single step, allowing them to run simultaneously. 
+
+The same applies to data writers. Also, transformations independent of the connectors can be grouped within the same step. Note that this does not reduce costs, as each job consume credits independently.
+
+***Warning:** If too many tasks are scheduled in a single phase, you may exceed the available Storage job slots, causing delays in your flow’s execution. Limiting the number of concurrent component jobs 
+to 10 is recommended to avoid reaching Storage capacity limits. You can, of course, configure your flows to execute more jobs in parallel. Keboola will then concurrently execute the jobs to the maximum 
```

Comment:
"Limiting the number of concurrent component jobs 
to 10 is recommended" to jako není úplně pravda, ale asi abych to tu nechal, nevím jak to jednodušeji vysvětlit